innovation.game.php

```diff
@@ -10129,6 +10130,7 @@ function stPlayerInvolvedTurn() {
                 // Put them back in hand so the return order can be determined by the player
                 $cards = self::getCardsInLocation($player_id, 'revealed');
                 foreach ($cards as $card) {
+                    $card = self::getCardInfo($card['id']);
```

Comment:
This bug doesn't actually happen when the card is in a publicly visible location (e.g. revealed or board). So let's remove this line so that we aren't making an unnecessary SQL query.